server_admin/topics/users/delete-user.adoc

```diff
@@ -1,19 +1,14 @@
-[[_delete-user]]
-
-=== Deleting Users
-
-To delete a user click on `Users` in the left menu bar.
-
-.Users
-image:{project_images}/users.png[]
-
-This menu option brings you to the user list page. Click `View all users` or search to find the user you intend to delete.
-
-.View All Users
-image:{project_images}/delete-user.png[]
-
-In the list of users, click `Delete` next to the user you want to remove. You will be asked to confirm that you are sure you want to delete this user. Click `Delete` in the confirmation box to confirm.
+[deleting-users]
+= Deleting Users
 
+To remove a user from the console you must delete the user.
 
+.Prerequisites
+* You are logged in to the correct realm and are not in the master realm.
```

Comment:
Why is it important to not be in the master realm? Also, please have a discussion with dev about using the term master. Can we change it to main?

Reply:
Stian is working on changing master to an acceptable term.  He is working with the community on this.  The terms master, blacklist, and white list need to change in the code first.